helm/idol-nifi/templates/nifi-registry/nifi-registry.yaml

```
@@ -41,7 +42,7 @@ spec:
       imagePullSecrets:
       - name: dockerhub-secret
       containers:
-      - name: nifi-registry
+      - name: {{ .Values.name }}-reg
```

```suggestion
      - name: nifi-registry
```
Container name is only used internally within the pod so I think we should leave this as nifi-registry for clarity.